hooks/src/internal.d.ts

```diff
@@ -7,9 +7,11 @@ import { Reducer } from '.';
 
 export { PreactContext };
 
-export type Internal = PreactInternal & {
-	_component: Component;
-};
+export interface Internal extends PreactInternal {
+	data: PreactInternal.data & {
+		__hooks: ComponentHooks;
```

Comment:
We could probably have this live in the base Internal interface.

Reply:
Don't mind really having it here. If we're super strict about our addons it's the right place to put it here.